packs/bit-bucket/actions/lib/action.py

```diff
@@ -0,0 +1,17 @@
+from st2actions.runners.pythonrunner import Action
+from bitbucket.bitbucket import Bitbucket
+
+
+class BitBucketAction(Action):
+    def __init__(self, config):
+        super(BitBucketAction, self).__init__(config)
+
+    def perform_request(self, repo=''):
```

Comment:
This method name is slightly confusing since it doesn't perform an actual http request - it just instantiates a client.

Calling it `_get_client` or similar would be less confusing, imo.